themes/digital.gov/layouts/partials/core/get_authors_short.html

```diff
@@ -26,10 +26,10 @@
       {{/* Lets look up the author data, based on the UID. Should equal the filename */}}
       {{- with $.Site.GetPage (printf "/%s/%s" $taxonomy $uid) -}}
 
-      <a class="author" href="{{- (printf "authors/%s/" $uid) | absURL -}} " title="Posts by {{- .Params.display_name | default $uid -}} " rel="author">
+      <a class="author" href="{{- (printf "authors/%s/" $uid) | absURL -}} " title="Posts by&nbsp;{{- .Params.display_name | default $uid -}} " rel="author">
```

Comment:
Having the title and the `usa-sr-only` text inside could cause it to be read twice on some screen readers.

I would remove one.

Reply:
I have remove the `title` attribute and kept the `usa-sr-only` span.